### PR TITLE
Upgrade to version 0.0.28

### PR DESCRIPTION
**Pull request by dmeecs**
API version: 1.275.0
Package version: 0.0.28

### CODE REVIEW

**Review by MrTam — APPROVED** [review]

**Review by chrisimcevoy — CHANGES_REQUESTED** [review]
Version should be 0.0.28, not 0.0.15.
Will update this shortly.

File: model_maven_package_upload_request.go [hunk 1]
```diff
@@ -21,6 +21,8 @@ type MavenPackageUploadRequest struct {
 	ArtifactId NullableString `json:"artifact_id,omitempty"`
 	// Artifact's group ID.
 	GroupId NullableString `json:"group_id,omitempty"`
+	// The ivy file is an XML file describing the dependencies of the project.
+	IvyFile NullableString `json:"ivy_file,omitempty"`
```
Comment (chrisimcevoy) [inline model_maven_package_upload_request.go:25]: Looks to be the same as pom file, so... yeah.

File: README.md [hunk 2]
```diff
@@ -5,8 +5,8 @@ The API to the Cloudsmith Service
 ## Overview
 This API client was generated by the [OpenAPI Generator](https://openapi-generator.tech) project.  By using the [OpenAPI-spec](https://www.openapis.org/) from a remote server, you can easily generate an API client.
 
-- API version: 1.273.0
-- Package version: 0.0.14
+- API version: 1.275.0
+- Package version: 0.0.28
```
Comment (chrisimcevoy) [inline README.md:9]: Versions as expected now

File: model_package_tag_request.go [hunk 3]
```diff
@@ -30,7 +30,7 @@ type PackageTagRequest struct {
 // will change when the set of required properties is changed
 func NewPackageTagRequest() *PackageTagRequest {
 	this := PackageTagRequest{}
-	var action ACTION = "Add"
+	var action string = "Add"
```
Comment (chrisimcevoy) [inline model_package_tag_request.go:33]: This must have been broken recently!

File: model_repository_token.go [hunk 4]
```diff
@@ -80,10 +80,10 @@ type RepositoryToken struct {
 // will change when the set of required properties is changed
 func NewRepositoryToken(name string) *RepositoryToken {
 	this := RepositoryToken{}
-	var limitBandwidthUnit LIMIT_BANDWIDTH_UNIT = "Byte"
```
Comment (chrisimcevoy) [inline model_repository_token.go:83]: This must have been broken recently too... fixed now.

**Review by chrisimcevoy — APPROVED** [review]

**Pull request merged**